contracts/PublicResolver.sol

```diff
@@ -52,4 +53,14 @@ contract PublicResolver is ABIResolver, AddrResolver, ContentHashResolver, DNSRe
         address owner = ens.owner(node);
         return owner == msg.sender || authorisations[node][owner][msg.sender];
     }
+
+    function multicall(bytes[] calldata datas) external returns(bytes[] memory rets) {
```

Comment:
the plural of data is data.

Reply:
Yeah, but then we have no variable name to use for the singular.

Reply:
I don't see dada used anywehre?

Reply:
But if we did I'd have to rename this one.

Is this important?

Reply:
Singular of `data` is `datum`

Reply:
or you can call `arrayofdata` a bit long though